src/MicrosoftTeams.schema.json

```diff
@@ -247,38 +248,18 @@
                                         "properties": {
                                             "title": {
                                                 "type": "string",
-                                                "description": "The one word command name",
-                                                "maxLength": 16
-                                            },
-                                            "usageHint": {
-                                                "type": "string",
-                                                "description": "A short usage template for the command",
-                                                "maxLength": 32
-                                            },
-                                            "shortDescription": {
-                                                "type": "string",
-                                                "description": "A short description of the command",
+                                                "description": "The bot command name",
                                                 "maxLength": 32
```

Comment:
Larry and I discussed about the max length of the command name and description and we came out these numbers. Haven't got UX redline yet. Let's keep them for the moment.